libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -2788,6 +2802,8 @@ const char *AP_Mission::Mission_Command::type() const
         return "WP";
     case MAV_CMD_NAV_SPLINE_WAYPOINT:
         return "SplineWP";
+    case MAV_CMD_NAV_ARC_WAYPOINT:
+        return "ArcWP";
```

Comment:
I don't think you need the "WP", just "Arc" is OK.

Reply:
I agree.